README.md

```diff
@@ -40,7 +40,7 @@ Navigate to Concierge's root directory (in the directory that contains main.js)
     }
 }
 ```
-Replace each of the angle bracketed strings (`<...>`) with the respective information hinted inside the brackets.
+Replace each of the angle bracketed strings (`<...>`) with the respective information hinted inside the brackets. The `commandPrefix` is a textual marker. Any messages meant for Concierge must begin with the `commandPrefix`.
```

Comment:
"should" rather than "must". There is always going to be cases where the `commandPrefix` will not be appropriate. E.g. associate